osu.Game/Graphics/UserInterface/HoverSounds.cs

```diff
@@ -24,35 +25,39 @@ public class HoverSounds : CompositeDrawable
         /// <summary>
         /// Length of debounce for hover sound playback, in milliseconds. Default is 50ms.
         /// </summary>
-        public double HoverDebounceTime { get; } = 50;
+        public double HoverDebounceTime { get; } = 20;
 
         protected readonly HoverSampleSet SampleSet;
 
+        private Bindable<double> lastPlaybackTime;
+
         public HoverSounds(HoverSampleSet sampleSet = HoverSampleSet.Normal)
         {
             SampleSet = sampleSet;
             RelativeSizeAxes = Axes.Both;
         }
 
-        private ScheduledDelegate playDelegate;
+        [BackgroundDependencyLoader]
+        private void load(AudioManager audio, SessionStatics statics)
+        {
+            lastPlaybackTime = statics.GetBindable<double>(Static.LastHoverSoundPlaybackTime);
+
+            sampleHover = audio.Samples.Get($@"UI/generic-hover{SampleSet.GetDescription()}");
+        }
 
         protected override bool OnHover(HoverEvent e)
         {
-            playDelegate?.Cancel();
+            bool requiresDebounce = HoverDebounceTime <= 0;
```

Comment:
this is backwards, isn't it? also truth be told I don't think this needs to exist, really - always testing against `HoverDebounceTime` even if it's zero/negative should suffice

Reply:
Good catch, was added as an afterthought and was broken.